src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImpl.java

```diff
@@ -129,6 +131,11 @@ public RassXmlProcessingResults updateKFS(List<RassXmlFileParseResult> successfu
                 List<?> xmlObjects = (List<?>) ObjectPropertyUtils.getPropertyValue(
                         documentWrapper, objectDefinition.getRootXmlObjectListPropertyName());
                 
+                if (Agency.class.getSimpleName().equals(objectDefinition.getObjectLabel())) {
+                    LOG.info("updateBOs, found a collection of Agencies, need to do our custom sort.");
```

Comment:
Since we do use a log statement here I wonder if we should add a bit more info on what is being done?